lib/engine/spender.rb

```diff
@@ -7,6 +7,7 @@ module Spender
     attr_accessor :cash
 
     def check_cash(amount)
+      amount.bad_func if (@cash - amount).negative?
```

Comment:
bad_func ????

Reply:
Debug code because the game error wouldn't tell me which action was causing the issue for the above json. Needs to be removed!